themes/nekolight/nekolight.theme.sh

```diff
@@ -0,0 +1,27 @@
+#! bash oh-my-bash.module
+
+export _omb_prompt_nekolight_version='1.0.0'
```

Comment:
```suggestion
_omb_prompt_nekolight_version='1.0.0'
```

This doesn't need to be exported. The export attribute is used when external programs started in the shell session want to reference its value.

Reply:
Also the variable name should start with `_omb_theme_nekolight` (but not `_omb_prompt_nekolight_`).